build_runner_core/lib/src/generate/build_definition.dart

```diff
@@ -197,15 +204,6 @@ class _Loader {
       assetGraph,
     );
     updates.addAll(_computeBuilderOptionsUpdates(assetGraph, buildPhases));
-    await assetGraph.updateAndInvalidate(
```

Comment:
It is non-obvious to me why this is not needed any more?

Reply:
(answering my own question: digging around I found that the updates are passed as `updatesFromLoad` for `BuildSeries` and that on `BuildSeries.run`s first run these are passed to Build.run, which runs _safeBuild with them that call _updateAssetGraph with them which calls assetGraph.updateAndInvalidate...)

Reply:
Right :)